package.json

```diff
@@ -3,11 +3,12 @@
   "dependencies": {
     "@koa/cors": "^3.0.0",
     "@koa/router": "^8.0.2",
+    "format-link-header": "^2.1.0",
```

Comment:
Opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40411 for typing.

Reply:
FYI `set-link` now also has typings ;)